src/DurableTask.Netherite/TransportLayer/EventHubs/PartitionProcessor.cs

```diff
@@ -92,70 +95,81 @@ public EventHubsProcessor(
+            {
+                return;
+            }
+
             List<BlockBlobClient> obsoleteBatches = null;
 
             if (this.traceHelper.IsEnabled(LogLevel.Trace))
             {
-                this.traceHelper.LogTrace("EventHubsProcessor {eventHubName}/{eventHubPartition} confirmed event nextInputQueuePosition={nextInputQueuePosition}", this.eventHubName, this.eventHubPartition, ((PartitionEvent)evt).NextInputQueuePositionTuple);
+                this.traceHelper.LogTrace("EventHubsProcessor {eventHubName}/{eventHubPartition} ConfirmDurable nextInputQueuePosition={nextInputQueuePosition}", this.eventHubName, this.eventHubPartition, ((PartitionEvent)evt).NextInputQueuePositionTuple);
             }
 
             // this is called after an event has committed (i.e. has been durably persisted in the recovery log).
-            // so we know we will never need to deliver it again. We remove it from the local buffer, update the fields that
-            // track the last persisted position, and delete the blob batch if this was the last event in the batch.
+            // so we know we will never need to deliver it again. We remove it from the local buffer
+            // and delete the blob batch if this was the last event in the batch.
             while (this.pendingDelivery.TryPeek(out var front) 
                 && (front.Event == evt || front.Event.NextInputQueuePositionTuple.CompareTo(((PartitionEvent) evt).NextInputQueuePositionTuple) < 0))
             {
                 if (this.pendingDelivery.TryDequeue(out var confirmed) && front == confirmed)
                 {
                     if (this.traceHelper.IsEnabled(LogLevel.Trace))
                     {
-                        this.traceHelper.LogTrace("EventHubsProcessor {eventHubName}/{eventHubPartition} discarding buffered event nextInputQueuePosition={nextInputQueuePosition} lastInBatch={lastInBatch} seqno={seqNo} offset={offset} batchBlob={batchBlob}", this.eventHubName, this.eventHubPartition, front.Event.NextInputQueuePositionTuple, confirmed.LastInBatch, confirmed.SeqNo, confirmed.Offset, confirmed.BatchBlob);
+                        this.traceHelper.LogTrace("EventHubsProcessor {eventHubName}/{eventHubPartition} discarding buffered event nextInputQueuePosition={nextInputQueuePosition} lastInBatch={lastInBatch} seqno={seqNo} batchBlob={batchBlob}", this.eventHubName, this.eventHubPartition, front.Event.NextInputQueuePositionTuple, confirmed.LastInBatch, confirmed.SeqNo, confirmed.BatchBlob?.Name);
                     }
 
                     if (confirmed.LastInBatch)
                     {
-                        // to create EH checkpoints we need to record both the EH sequence number and the EH offset (see SaveEventHubsReceiverCheckpoint)
-                        this.persistedOffset = Math.Max(this.persistedOffset, confirmed.Offset);
-                        this.persistedSequenceNumber = Math.Max(this.persistedSequenceNumber, confirmed.SeqNo);
-
```

Comment:
why isn't this needed anymore?

Reply:
with the new EH SDK, we are no longer forced to use the EH-built-in-checkpointing. Instead we can, at partition startup time. specify the position at which we would start to receive events from. 

This is much simpler for us since we already have that information stored in our partition state.  So we can avoid those unnecessary extra checkpoints that EH forced us to do.